test/run_test.py

```diff
@@ -105,6 +105,11 @@
 sys.stderr.write("Running poppunk_info\n")
 subprocess.run(python_cmd + " ../poppunk_info-runner.py --db example_db --output example_db.info.csv", shell=True, check=True)
 
+# lineages from strains
+sys.stderr.write("Running poppunk_lineages_from_strains\n")
+subprocess.run(python_cmd + " ../poppunk_lineages-runner.py --create-db --db example_db --db-scheme example_lineage_scheme.pkl --output lineage_creation_output", shell=True, check=True)
```

Comment:
Possible/sensible to check results, as well as that it can run?

Reply:
Do we want to make that standard for all these tests?

Reply:
(in that if there's a file path/IO error that affects all tests, it might only raise the alarm with this test if it is checked differently, which I would inevitably wrongly assume to be caused by the lineage model-specific code.

Reply:
I think it's good that we check that we can actually run with all of the options, but as we are settling on a more final version we should probably start trying to add more tests of results. Doing it all now probably too tedious so let's aim to add them slowly...